src/posts/server-running-p.rst

```diff
@@ -0,0 +1,69 @@
+((title . "Determining if the server is started, or the wonders of server-running-p")
+ (date . "2018-04-01 13:43:26 +02:00"))
+
+Trivia: How can you determine if the current Emacs instance has the
+Emacs server running?
+
+A quick search gives us three potential candidates: ``server-mode``,
+``(daemonp)`` and ``(server-running-p)``.  That's way too much, but
+surely one of them is the right one, isn't it?  Well, no.  Because the
+real answer to this trivial question is: *you can't*.
+
+ - ``server-mode`` is `t` if, and only if, the server was started
+using the function with the same name.  But there are other ways to
+run the server, like `M-x server-start` or `emacs --daemon`.
+
+ - ``(daemeonp)`` returns t if, and only if, Emacs was started in
+   daemon mode.
+
+What about `(server-running-p)`, then?  Well, it may look friendly,
```

Comment:
You need to double the backticks here and for everything after this, single backticks are for links.